CHANGELOG.md

```diff
@@ -7,25 +7,41 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [python-0.6.0] - 2024-01-23
+
 ### Added
 
-- Added `plot.state` function in pylace to render PCC states
-- Added `analysis.explain_prediction` in pylace to explain predictions
-- Added `plot.prediction_explanation` in pylace to render prediction explanations
-- Added `analysis.held_out_uncertainty` in pylace
-- Added `analysis.attributable_[neglogp | inconsistrncy | uncertainty]` in pylace to quantify the amount of surprisal (neglogp), inconsistency, and uncertainty attributable to other features
+- Added support for Python 3.12
+- Added `plot.state` function to render PCC states
+- Added `analysis.explain_prediction` to explain predictions
+- Added `plot.prediction_explanation` to render prediction explanations
+- Added `analysis.held_out_uncertainty`
+- Added `analysis.attributable_[neglogp | inconsistrncy | uncertainty]` to quantify the amount of surprisal (neglogp), inconsistency, and uncertainty attributable to other features
+- Added `utils.predict_xs`
+
+### Changed
+
```

Comment:
Do we want to mention the dependency updates too?

Reply:
I waffled on that, so if you thought of it then I suppose we should

Reply:
I've added the updated dependencies. I tried to list the ones that I thought an end user would have to care about, instead of all the ones that were changed; let me know if you think that's a bad decision